udaykalra/portfolio/target/portfolio-1/style.css

```diff
@@ -1,9 +1,144 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.bottomRow
+{
+	padding-top: 30px;
 }
-
-#greeting-container {
-  margin-top: 20px;
+.center
+{
+	margin: auto;
+	text-align: center;
 }
+.hero-image
+{
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+ url('hero.jpg');
+	background-position: center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	height: 50%;
+	position: relative;
+}
+.hero-text
+{
+	color: white;
+	font: 'Roboto', sans-serif;
+	font-size: 40px;
+	left: 50%;
+	position: absolute;
+	text-align: center;
+	top: 50%;
+	transform: translate(-50%, -50%);
+}
+.hero-text h1
+{
+	font-size: 60px;
+}
+.textbox
+{
+	background: var(--light-color);
+	border: 2px solid var(--light-color);
+	border-radius: 15px;
+}
+.textbox h1
+{
+	font-size: 50px;
+}
+.textbox p
+{
+	font-size: 30px;
+}
+.topRow
+{
+	padding-top: 30px;
+}
+:root
+{
+	--body-color: #121212;
+	--head-color: #000000;
+	--light-color: #333333;
+	--main-color: #fff;
+	--text-color: #e7e7e7;
+}
+body
+{
+	background: 100%;
+}
+div.animate a:after
+{
+	bottom: 0px;
+	transform: translateY(-10px);
+}
+div.animate a:before
+{
+	top: 0px;
+	transform: translateY(10px);
+}
+div.animate a:before,
+div.animate a:after
```

Comment:
Style encourages separating selectors with newlines:
https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation

Also applies to other rules in this file.

Reply:
This has been implemented in latest push.